new/LIT7057Fever.xml

```diff
@@ -0,0 +1,63 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7057Fever" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎተ፡ ፌራ፡ ወንዳድ፡ ዘቅዱስ፡ ሚካኤል፡ </title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalota ferā wa-nǝdād za-qǝddus Mikāʾel</title>
+                <title xml:lang="en" corresp="#t1">Prayer against contagious disease (ferā) and malarial fever (nədād) by the Archangel Michael</title>
```

Comment:
Is "by" intended to mean that the archangel wrote it? Maybe "of" would be more neutral?

Reply:
The word is from Strelcyn. I think, the idea is, that the Archangel recited it. If you disagree with that interpretation, I am open to change it.

<img width="698" alt="Screenshot 2024_Prayer fever" src="https://github.com/user-attachments/assets/e146de51-3565-490c-8b22-e53717d981cf">